scst-scan/install-prisma-integration.hbs.md

```diff
@@ -0,0 +1,566 @@
+
+### Supply Chain Security Tools - Store integration
+
+Using Supply Chain Security Tools - Store Integration: To persist the results found by the Prisma Scanner, you can enable the Supply Chain Security Tools - Store integration by appending the fields to the values.yaml file.
+
+The Grype, Snyk, and Prisma Scanner Integrations enable the Metadata Store. To prevent conflicts, the configuration values are slightly different based on whether the Grype Scanner Integration is installed or not. If Tanzu Application Platform is installed using the Full Profile, the Grype Scanner Integration is installed, unless it is explicitly excluded.
+
+If the Grype or Snyk Scanner Integration is installed in the same dev-namespace Prisma Scanner is installed:
+
+```
+#! ...
+metadataStore:
+  #! The url where the Store deployment is accesible.
+  #! Default value is: "https://metadata-store-app.metadata-store.svc.cluster.local:8443"
+  url: "<STORE-URL>"
+  caSecret:
+    #! The name of the secret that contains the ca.crt to connect to the Store Deployment.
+    #! Default value is: "app-tls-cert"
+    name: "<CA-SECRET-NAME>"
+    importFromNamespace: "" #! since both Prisma and Grype/Snyk both enable store, one must leave importFromNamespace blank
+  #! authSecret is for multicluster configurations.
+  authSecret:
+    #! The name of the secret that contains the auth token to authenticate to the Store Deployment.
+    name: "<AUTH-SECRET-NAME>"
+    importFromNamespace: "" #! since both Prisma and Grype/Snyk both enable store, one must leave importFromNamespace blank
+```
+
+If the Grype/Snyk Scanner Integration is not installed in the same dev-namespace Prisma Scanner is installed:
+
+```
+#! ...
+metadataStore:
+  #! The url where the Store deployment is accesible.
+  #! Default value is: "https://metadata-store-app.metadata-store.svc.cluster.local:8443"
+  url: "<STORE-URL>"
+  caSecret:
+    #! The name of the secret that contains the ca.crt to connect to the Store Deployment.
+    #! Default value is: "app-tls-cert"
+    name: "<CA-SECRET-NAME>"
+    #! The namespace where the secrets for the Store Deployment live.
+    #! Default value is: "metadata-store"
+    importFromNamespace: "<STORE-SECRETS-NAMESPACE>"
+  #! authSecret is for multicluster configurations.
+  authSecret:
+    #! The name of the secret that contains the auth token to authenticate to the Store Deployment.
+    name: "<AUTH-SECRET-NAME>"
+    #! The namespace where the secrets for the Store Deployment live.
+    importFromNamespace: "<STORE-SECRETS-NAMESPACE>"
+```
+
+**Without Supply Chain Security Tools - Store Integration:** If you don’t want to enable the Supply Chain Security Tools - Store integration, explicitly deactivate the integration by appending the following fields to the values.yaml file that is enabled by default:
+
+```
+# ...
+metadataStore:
+  url: "" # Configuration is moved, so set this string to empty
+```
+### Sample ScanPolicy for CycloneDX Format
```

Comment:
For some reason the format on the actual copy renders the following outside the code block:
```
apiVersion: scanning.apps.tanzu.vmware.com/v1beta1 kind: ScanPolicy metadata: name: scan-policy labels: 'app.kubernetes.io/part-of': 'enable-in-gui' spec: regoFile: | package main```

To fix: put a opening ``` and closing ``` for this scanpolicy code block

Reply:
done!